src/app/components/EasyReadCTA/index.tsx

```diff
@@ -0,0 +1,122 @@
+/** @jsx jsx */
+import { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import { GridItemMedium } from '#app/legacy/components/Grid';
+import { RequestContext } from '#app/contexts/RequestContext';
+import useClickTrackerHandler from '#app/hooks/useClickTrackerHandler';
+import useViewTracker from '#app/hooks/useViewTracker';
+import Text from '../Text';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import styles from './index.styles';
+import { defaultTranslations } from './config';
+import InlineLink from '../InlineLink';
+import { FormatIcon } from '../icons';
+
+type CtaLinkProps = {
+  href: string;
+  text: string;
+  selected?: boolean;
+};
+
+export const createHrefRelativeToPage = (currentPath: string, id?: string) => {
+  const noExtenstionPath = currentPath.split('.')[0];
+  const slugs = noExtenstionPath.split('/');
+  const idToAppend = id ?? slugs[slugs.length - 1];
+  slugs.splice(-1, 1, idToAppend);
+
+  return slugs.join('/').replace(' ', '');
+};
+
+const eventTrackingData = {
+  componentName: 'easyReadCta',
+};
+
+const CtaLink = ({ href, text, selected = false }: CtaLinkProps) => {
+  const clickTrackerHandler = useClickTrackerHandler(eventTrackingData);
+
+  return (
+    <a
+      href={href}
+      css={styles.linkContainer}
+      {...clickTrackerHandler}
+      {...(selected && { 'aria-current': 'page' })}
+    >
+      <span css={styles.linkTextContainer}>
+        <Text
+          size="brevier"
+          css={[
+            styles.linkText,
+            selected ? styles.selected : styles.notSelected,
+          ]}
+        >
+          {text}
+        </Text>
+      </span>
+    </a>
+  );
+};
+
+type Props = {
+  easyReadAssetId?: string;
+  originalAssetId?: string;
+};
+
+const EasyReadCTA = ({ easyReadAssetId, originalAssetId }: Props) => {
+ const viewTracker = useViewTracker(eventTrackingData);
+  const { pathname } = useContext(RequestContext);
+  const { translations } = useContext(ServiceContext);
+
+  if (easyReadAssetId == null && originalAssetId == null) {
+    return null;
+  }
+
+  const easyHref = createHrefRelativeToPage(pathname, easyReadAssetId);
+  const originalHref = createHrefRelativeToPage(pathname, originalAssetId);
+
+  const isEasyActive = easyReadAssetId == null;
+
+  const { easyReadSite = defaultTranslations } = translations;
+  const {
+    toStandardSite,
+    toEasySite,
+    format,
+    aIDisclaimer,
+    learnMore,
+    learnMoreLink,
+  } = easyReadSite;
+
+  const id = 'Format';
+
+  return (
+    <GridItemMedium>
+      <section
+        role="region"
+        data-e2e="easy-read-cta"
+        aria-labelledby={id}
+        {...viewTracker}
+      >
+        <FormatIcon css={styles.icon} />
+        <Text as="strong" id={id} hidden>
+          {format}
+        </Text>
+        <CtaLink
```

Comment:
Could we add the word 'read' as visually hidden text so this is annouced as 'Standard read'? Note we need to get a translation.

`<a href="url" aria-current="page">
  <span role="text">
    Standard <span class="visually-hidden">read</span>
  </span>
</a>`

Reply:
Would be "à lire" would be the french translation for 'read'

Reply:
Due to the complexities of internationalisation, we have decided to not use visually hidden text for the trial, and instead have the full text displayed 'Lecture standard' in French and 'Standard read' in English.